formula-android/src/main/java/com/instacart/formula/android/FragmentStoreBuilder.kt

```diff
@@ -1,45 +1,26 @@
 package com.instacart.formula.android
 
-import com.instacart.formula.android.internal.Binding
-import com.instacart.formula.android.internal.Bindings
-import com.instacart.formula.android.internal.FunctionUtils
 import com.instacart.formula.android.internal.FeatureBinding
+import com.instacart.formula.android.internal.MappedFeatureFactory
 import java.lang.IllegalStateException
 import kotlin.reflect.KClass
 
 /**
- * A class used by [FragmentFlowStore] and [Flow] to register [fragment keys][FragmentKey] and their
+ * A class used by [FragmentFlowStore] to register [fragment keys][FragmentKey] and their
  * feature factories.
  */
 class FragmentStoreBuilder<Component> {
     companion object {
-
         @PublishedApi
         internal inline fun <Component> build(
             init: FragmentStoreBuilder<Component>.() -> Unit
-        ): Bindings<Component> {
+        ): List<FeatureBinding<Component, *>> {
```

Comment:
Lets make sure to bump the version to note that this is not a backwards compatible change.